tools/perf.sh

```diff
@@ -30,17 +30,167 @@ fi
+                        t_gamma_n_m1 [8]  = 3.3554
+                        t_gamma_n_m1 [9]  = 3.2498
+                        t_gamma_n_m1 [10] = 3.1693
+                        t_gamma_n_m1 [11] = 3.1058
+                        t_gamma_n_m1 [12] = 3.0545
+                        t_gamma_n_m1 [13] = 3.0123
+                        t_gamma_n_m1 [14] = 2.9768
+                        t_gamma_n_m1 [15] = 2.9467
+                        t_gamma_n_m1 [16] = 2.9208
+                        t_gamma_n_m1 [17] = 2.8982
+                        t_gamma_n_m1 [18] = 2.8784
+                        t_gamma_n_m1 [19] = 2.8609
+                        t_gamma_n_m1 [20] = 2.8453
+                        t_gamma_n_m1 [21] = 2.8314
+                        t_gamma_n_m1 [22] = 2.8188
+                        t_gamma_n_m1 [23] = 2.8073
+                        t_gamma_n_m1 [24] = 2.7969
+                        t_gamma_n_m1 [25] = 2.7874
+                        t_gamma_n_m1 [26] = 2.7787
+                        t_gamma_n_m1 [27] = 2.7707
+                        t_gamma_n_m1 [28] = 2.7633
+                        t_gamma_n_m1 [29] = 2.7564
+                        t_gamma_n_m1 [30] = 2.75
+                        t_gamma_n_m1 [31] = 2.744
+                        t_gamma_n_m1 [32] = 2.7385
+                        t_gamma_n_m1 [33] = 2.7333
+                        t_gamma_n_m1 [34] = 2.7284
+                        t_gamma_n_m1 [35] = 2.7238
+                        t_gamma_n_m1 [36] = 2.7195
+                        t_gamma_n_m1 [37] = 2.7154
+                        t_gamma_n_m1 [38] = 2.7116
+                        t_gamma_n_m1 [39] = 2.7079
+                        t_gamma_n_m1 [40] = 2.7045
+                        t_gamma_n_m1 [41] = 2.7012
+                        t_gamma_n_m1 [42] = 2.6981
+                        t_gamma_n_m1 [43] = 2.6951
+                        t_gamma_n_m1 [44] = 2.6923
+                        t_gamma_n_m1 [45] = 2.6896
+                        t_gamma_n_m1 [46] = 2.687
+                        t_gamma_n_m1 [47] = 2.6846
+                        t_gamma_n_m1 [48] = 2.6822
+                        t_gamma_n_m1 [49] = 2.68
+                        t_gamma_n_m1 [50] = 2.6778
+
+                        #
+                        # Sort array of measurements
+                        #
+                        for (i = 2; i <= n; i++) {
+                          j = i
+                          k = a [j]
+                          while (j > 1 && a [j - 1] > k) {
+                            a [j] = a [j - 1]
+                            j--
+                          }
+                          a [j] = k
+                        }
+
+                        #
+                        # Remove 20% of lowest and 20% of highest values
+                        #
+                        n_20_percent = int (n / 5)
+
+                        for (i = 1; i <= n_20_percent; i++) {
+                          delete a[n]
```

Comment:
On the 2nd and further iterations we try to delete elements, which do not exist, don't we?